### PR TITLE
Loader - New Component

### PR DESCRIPTION
**Pull request by anechol**
# Description

Adds new Loader component.

[DSS-442](https://kajabi.atlassian.net/browse/DSS-442)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests you've added and run to verify your changes.
Provide instructions so that we can reproduce.
Please also list any relevant details for your test configuration.

- [x] unit tests
- [x] e2e tests
- [x] accessibility tests
- [x] tested manually
- [ ] other:

**Test Configuration**:

- Pine versions:
- OS:
- Browsers: Safari, Chrome, Firefox
- Screen readers: Apple Voiceover
- Misc:

# Checklist:

If not applicable, leave options unchecked.

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

[DSS-442]: https://kajabi.atlassian.net/browse/DSS-442?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/core/src/components/pds-loader/pds-loader.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+  /**
+   * Determines whether the loader should display a label.
+   */
+  @Prop() showLabel?: boolean = false;
+
+  /**
+   * Sets the size of the spinner loader. Value can be preset or custom.
+   */
+  @Prop({ reflect: true }) size?: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | string = 'md';
```
Comment (pixelflips) [inline libs/core/src/components/pds-loader/pds-loader.tsx]: Would be great to see these presets documented.

**Review by QuintonJason — APPROVED** [review]
This is amazing work!

**Review by pixelflips — APPROVED** [review]
One minor docs comment, otherwise LGTM! Nice work! 👍🏼

File: libs/core/src/components/pds-loader/pds-loader.tsx (hunk 1 above)
Comment (anechol) [reply libs/core/src/components/pds-loader/pds-loader.tsx]: Got this in. Thanks for pointing this out.

**Pull request merged**